app/models/item.rb

```diff
@@ -0,0 +1,10 @@
+class Item < ApplicationRecord
+  validates :name, presence: true
+  validates :description, presence: true
+  validates :merchant_id, presence: true
+  validates :unit_price_in_cents, presence: true
+  validates :updated_at, presence: true
+  validates :created_at, presence: true
```

Comment:
You could get these all on one line by doing `validates_presence_of :name, :description...` instead.